package.json

```diff
@@ -4,17 +4,18 @@
   "dependencies": {
     "@rails/ujs": "^6.0.0-alpha",
     "@rails/webpacker": "^4.0.7",
-    "css-loader": "^3.1.0",
+    "axios": "^0.19.0",
+    "css-loader": "^3.2.0",
```

Comment:
noticed updated `package.json` but there's no `package-lock.json` file anywhere
are you in the "package-locks are the worst" camp? fair enough

Reply:
I'm using yarn, not npm, so it's in the yarn.lock